crates/contracts/build.rs

```diff
@@ -19,16 +19,7 @@ fn main() {
     println!("cargo:rerun-if-changed=build.rs");
 
     generate_contract_with_config("CoWSwapEthFlow", |builder| {
-        builder
-            .contract_mod_override("cowswap_eth_flow")
```

Comment:
We decided to deploy 1 contract for staging and 1 for prod. Hence, I don't import contract address as before, but rather via env variables